pkg/sql/alter_table.go

```diff
@@ -280,6 +280,7 @@ func (n *alterTableNode) startExec(params runParams) error {
 				idx := descpb.IndexDescriptor{
 					Name:             string(d.Name),
 					Unique:           true,
+					Invisible:        false,
```

Comment:
Naming consistency: consider renaming the field in the index descriptor proto to `hidden`.